backend/functions/lib/adapters/gateways/dxtr/case-docket.dxtr.gateway.test.ts

```diff
@@ -8,6 +8,7 @@ import {
   DXTR_CASE_DOCKET_ENTRIES,
   DXTR_DOCKET_ENTRIES_DOCUMENTS,
 } from '../../../testing/mock-data/case-docket-entries.mock';
+import { NOT_FOUND_ERROR_CASE_ID } from '../../../cosmos-humble-objects/fake.cosmos-client-humble';
```

Comment:
This import feels out of place. Maybe `NOT_FOUND_ERROR_CASE_ID` should be defined in a more common place? The fake cosmos client has nothing to do with docket.

Reply:
It also looks like we've got a copy of this constant in `backend/functions/lib/adapters/gateways/dxtr/case-docket.mock.gateway.ts` which would be better to use.

Reply:
agreed.